### PR TITLE
chore: enable building utils files

### PR DESCRIPTION
**Pull request by DominicKramer**

### CODE REVIEW

**Comment by DominicKramer** [comment]
PTAL

File: utils/fuzzer.ts [hunk 1]
```diff
@@ -0,0 +1,298 @@
+/**1
```
Comment (kjin) [inline utils/fuzzer.ts]: Nit: typo

File: utils/fuzzer.ts [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+ */
+// jscs:disable
+// jshint ignore: start
+
+'use strict';
+var indexOf = require('lodash.indexof');
+var without = require('lodash.without');
+var maxBy = require('lodash.maxby');
+var random = require('lodash.random');
+var is1 = require('is');
```
Comment (kjin) [inline utils/fuzzer.ts]: Why `is1`?

File: utils/fuzzer.ts [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+      allowedDepth
+    ) {
+      var lenChecked = isNumber(len) ? len : random(1, 10);
+      var availableTypes =
+        isString(ofOneType) && indexOf(this.types(), ofOneType) > -1
+          ? [ofOneType]
+          : this.types();
+      var currentDepthChecked = isNumber(currentDepth) ? currentDepth : 0;
+      var allowedDepthChecked = isNumber(allowedDepth) ? allowedDepth : 3;
+      var arr: {}[] = [];
```
Comment (kjin) [inline utils/fuzzer.ts]: Nit: Prefer Array<{}> here. We prefer Array\<T\> unless the type is just alphanumeric + dot.

File: utils/fuzzer.ts (hunk 1 above)
Comment (DominicKramer) [reply utils/fuzzer.ts]: Fixed

File: utils/fuzzer.ts (hunk 3 above)
Comment (DominicKramer) [reply utils/fuzzer.ts]: Fixed

File: utils/fuzzer.ts (hunk 2 above)
Comment (DominicKramer) [reply utils/fuzzer.ts]: I switched it to `is`.

**Pull request merged**